compositor_pipeline/src/pipeline/mp4/mp4_input.rs

```diff
@@ -64,12 +67,15 @@ impl Mp4Input {
             Some(track) => {
                 let (sender, receiver) = crossbeam_channel::bounded(10);
                 let handle = match track.decoder_options() {
-                    DecoderOptions::H264(extra_data) => {
-                        spawn_video_decoder_thread::<ffmpeg_h264::FfmpegH264Decoder, 5, _>(
-                            ctx.clone(),
-                            input_id.clone(),
-                            Some(AvccToAnnexBRepacker::new(extra_data.clone())),
-                            sender,
+                    DecoderOptions::H264(h264_config) => {
+                        spawn_thread::<VideoDecoderThread<ffmpeg_h264::FfmpegH264Decoder, _>>(
+                            &input_id.0,
```

Comment:
whatever you are passing here should be transformed in the VideoDecoderThread code to add context that this is an input